components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,464 @@
+
+## Requirements
+
+The requirements in the [Omega-0 design document](OmegaV0ShallowWater) still apply. Additional design requirements for Omega-1 are:
+
+### Omega will be an incompressible, hydrostatic, non-Boussinesq ocean model.
+See discussion in introduction. Non-Boussinesq changes the independant vertical variable to pressure, and mass conservation equation to be pressure-thickness, $h=\Delta p$.
+
+### Omega will use TEOS10 for the equation of state.
+See additional EOS design document.
+
+### Omega-1.0 will add new terms for the pressure gradient, vertical mixing, and vertical advection.
+See individual design documents for each term.
+
+## Governing Equations
+
+### MPAS-Ocean Equations of Motion
+
+The MPAS-Ocean layered formulation solves the following equations for momentum, thickness, and tracers at layer $k$. These are continuous in the horizontal and discrete in the vertical.
+
+$$
+\frac{\partial {\mathbf u}_k}{\partial t}
++ \frac{1}{2}\nabla \left| {\mathbf u}_k \right|^2
++ ( {\mathbf k} \cdot \nabla \times {\mathbf u}_k) {\mathbf u}^\perp_k
++ f{\mathbf u}^{\perp}_k
++ w_k^{bot}{\mathbf u}_k^{bot}
+- w_k^{top}{\mathbf u}_k^{top} =
+- \frac{1}{\rho_0}\nabla p_k 
+- \frac{\rho g}{\rho_0}\nabla z^{mid}_k
++ \nu_h\nabla^2{\mathbf u}_k
++ \frac{\partial }{\partial z} \left( \nu_v \frac{\partial {\mathbf u}_k}{\partial z} \right),
+$$ (mpaso-continuous-momentum)
+
+$$
+\frac{\partial h_k}{\partial t} + \nabla \cdot \left( h_k^e {\mathbf u}_k \right) + w_k^{bot} - w_k^{top} = 0,
+$$ (mpaso-continuous-thickness)
+
+$$
+\frac{\partial h_k\varphi_k}{\partial t} + \nabla \cdot \left( h_k^e\varphi_k^e {\mathbf u}_k \right) 
++ \varphi_k^{bot} w_k^{bot} - \varphi_k^{top} w_k^{top}
+= \nabla\cdot\left(h_k^e \kappa_h \nabla\varphi_k \right)
++ h_k \frac{\partial }{\partial z} \left( \kappa_v \frac{\partial \varphi_k}{\partial z} \right).
+$$ (mpaso-continuous-tracer)
+
+The layer thickness $h$, vertical velocity $w$, pressure $p$, and tracer $\varphi$, are cell-centered quantities, while the horizontal velocity ${\mathbf u}$ and $e$ superscript are variables interpolated to the cell edges.
+
+### Continuous Equations
+The continuous form of the conservation equations are as follows [Kundu et al. 2024](https://www.amazon.com/Fluid-Mechanics-Pijush-K-Kundu/dp/012405935X), chapter 4, eqns 4.7 and 4.22. This is before any assumptions are made, so this is a compressible, non-hydrostatic, non-Boussinesq fluid. Here all variables are a function of $(x,y,z)$, $\bf u$ denotes the horizontal velocity vector, $w$ is the vertical velocity, ${\bf u} \otimes {\bf u} = {\bf u}{\bf u}^T$ is the tensor product, $\nabla$ is the horizontal gradient, and other variables defined in the [section below](#variable-definitions).
```

Comment:
can you name the section below -- 'in the variable definition section below'

Reply:
done.